common/oatbox/event/EventManager.php

```diff
@@ -53,7 +53,7 @@ public function trigger($event, $params = [])
                 list($key, $function) = $callback;
                 if (is_string($key)) {
                     try {
-                        $service = $this->getServiceManager()->get($key);
+                        $service = $this->getServiceManager()->getContainer()->get($key);
```

Comment:
🚀  Thanks a lot!

Reply:
Can I recommend to declare the $container outside of the loop to performance increasing?

```
$container = $this->getServiceManager()->getContainer();
```

```suggestion
                        $service = $container->get($key);
```

Reply:
Great idea! Updated :)